fulp_modules/bloodsuckers/bloodsucker/bloodsucker.dm

```diff
@@ -0,0 +1,746 @@
+//
+//	* Bloodsuckers are IMMORTAL
+//		- Brute damage cannot destroy them (and it caps very low, so they don't stack too much)
+//		- Burn damage can only kill them at very high amounts.
+//		- Removing the head kills the vamp forever.
+//		- Removing the heart kills the vamp until replaced.
+//
+//	* Bloodsuckers are DEAD
+//		- They do not breathe.
+//		- Cold affects them less.
+//		- They are immune to disease (but can spread it)
+//		- Food is useless and cause sickness.
+//		- Nothing can heal the vamp other than his own blood.
+//
+//	* Bloodsuckers are PREDATORS
+//		- They detect life/heartbeats nearby.
+//		- They know other predators instantly (Vamps, Werewolves, and alien types) regardless of disguise.
+//
+//
+//
+// 	* Bloodsuckers enter Torpor when DEAD or RESTING in coffin
+//		- Torpid vampires regenerate their health. Coffins negate cost and speed up the process.
+//		** To rest in a coffin, either SLEEP or CLOSE THE LID while you're in it. You will be given a prompt to sleep until healed. Healing in a coffin costs NO blood!
+//
+
+
+
+// 					O B J E C T I V E S
+//
+//
+//
+//
+//	1) GROOM AN HEIR:	Find a person with appropriate traits (hair, blood type, gender) to be turned as a Vampire. Before they rise, they must be properly trained. Raise them to great power after their change.
+//
+//	2) BIBLIOPHILE:		Research objects of interest, study items looking for clues of ancient secrets, and hunt down the clues to a Vampiric artifact of horrible power.
+//
+//	3) CRYPT LORD:		Build a terrifying sepulcher to your evil, with servants to lavish upon you in undeath. The trappings of a true crypt lord come at grave cost.
+//
+//	4) GOURMOND:		Oh, to taste all the delicacies the station has to offer! DRINK ## BLOOD FROM VICTIMS WHO LIVE, EAT ## ORGANS FROM VICTIMS WHO LIVE
+
+
+//			Vassals
+//
+// - Loyal to (and In Love With) Master
+// - Master can speak to, summon, or punish his Vassals, even while asleep or torpid.
+// - Master may have as many Vassals as they want
+
+
+
+// 			Dev Notes
+//
+// HEALING: Maybe Vamps metabolize specially? Like, they slowly drip their own blood into their system?
+//			- Give Vamps their own metabolization proc, perhaps?
+//			** shadowpeople.dm has rules for healing.
+//
+// KILLING: It's almost impossible to track who someone has directly killed. But an Admin could be given
+//			an easy way to whip a Bloodsucker for cruel behavior, as a RP mechanic but not a punishment.
+//			**
+//
+// HUNGER:  Just keep adjusting mob's nutrition to Blood Hunger level. No need to cancel nutrition from eating.
+//			** mob.dm /set_nutrition()
+//			** snacks.dm / attack()  <-- Stop food from doing anything?
+
+// ORGANS:  Liver
+//			** life.dm /handle_liver()
+//
+// CORPSE:	Most of these effects likely go away when using "Masquerade" to appear alive.
+//			** status_procs.dm /adjust_bodytemperature()
+//			** traits.dm /TRAIT_NOBREATH /TRAIT_SLEEPIMMUNE /TRAIT_RESISTCOLD /TRAIT_RADIMMUNE  /TRAIT_VIRUSIMMUNE
```

Comment:
won't TRAIT_VIRUSIMMUNE make them immune to contracting a virus in the first place?